strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileAttributes.java

```diff
@@ -1,14 +1,24 @@
 package org.carlspring.strongbox.providers.io;
 
-import static org.carlspring.strongbox.providers.io.RepositoryFileAttributeType.*;
+import org.carlspring.strongbox.artifact.coordinates.ArtifactCoordinates;
 
 import java.net.URL;
 import java.nio.file.attribute.BasicFileAttributes;
 import java.nio.file.attribute.FileTime;
 import java.util.HashMap;
 import java.util.Map;
 
-import org.carlspring.strongbox.artifact.coordinates.ArtifactCoordinates;
+import static org.carlspring.strongbox.providers.io.RepositoryFileAttributeType.ARTIFACT;
+import static org.carlspring.strongbox.providers.io.RepositoryFileAttributeType.ARTIFACT_PATH;
```

Comment:
I think this should be replaced with a wildcard `static import`. There's no point in listing them out one by one, if they're more than 3-4... 5, right...?

Reply:
I'm using `class count to use import with '*':` equal to `999` (which is close to "never use wildcard" :) ). What's the expected value ?

Reply:
Hahaha! :)
I think that anything above five static imports from the same class is probably too much... What do you reckon?